### PR TITLE
feat: add write method

### PR DESCRIPTION
**Pull request by Rachelint**
add write method in python client.

### CODE REVIEW

File: src/model.rs [hunk 1]
```diff
@@ -209,16 +226,248 @@ impl QueryResponse {
...
+pub struct ValueBuilder;
+
+#[pymethods]
+impl ValueBuilder {
+    #[new]
+    pub fn new() -> Self {
+        Self::default()
+    }
+
+    pub fn with_i8(&self, val: i8) -> Value {
```
Comment (ShiKaiWi) [inline src/model.rs:266]: I'm not sure whether we should remove the prefix `with_`? Is there any reference suggesting this style in python? I guess we'd better remove the prefix if not.

Comment (ShiKaiWi) [reply src/model.rs:266]: I see. The method names may be a conflict with rust's keywords if remove the prefix.

File: src/model.rs [hunk 2]
```diff
@@ -209,16 +226,248 @@ impl QueryResponse {
...
+
+    pub fn metric(&mut self, metric: String) {
+        self.metric = Some(metric);
+    }
+
+    pub fn timestamp(&mut self, timestamp: TimestampMs) {
+        self.timestamp = Some(timestamp);
+    }
+
+    pub fn tag(&mut self, name: String, val: Value) {
```
Comment (ShiKaiWi) [inline src/model.rs:380]: I guess we need `tags` method for caller to set all the tags at once.

**Review by ShiKaiWi — APPROVED** [review]
LGTM

**Comment by ShiKaiWi** [comment]
@Rachelint Maybe we should provide an example to show how to use the write method.

File: src/model.rs [hunk 3]
```diff
@@ -209,16 +226,248 @@ impl QueryResponse {
         }
     }
 
+    pub fn get_affected_rows(&self) -> u32 {
```
Comment (messense) [inline src/model.rs:229]: I think this can just be a getter by using `#[pyo3(get)]` on the field in struct definition?

Comment (Rachelint) [reply src/model.rs:229]: Thanks, done.

**Comment by Rachelint** [comment]
> @Rachelint Maybe we should provide an example to show how to use the write method.

I write the example both rust and python actually, planning to push them after.

File: src/model.rs (hunk 3 above)
Comment (ShiKaiWi) [reply src/model.rs:229]: I thought this had been fixed... @Rachelint Maybe you can create another PR to push the commit about this refactor.

**Pull request merged**